sonata-project/page-bundle/3.9/src/Entity/SonataPageBlock.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as Serializer;
+use Sonata\PageBundle\Entity\BaseBlock;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="page__block")
+ * @ORM\HasLifecycleCallbacks
+ */
+class SonataPageBlock extends BaseBlock
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * // Serializer\Groups(groups={"sonata_api_read","sonata_api_write","sonata_search"})
+     *
+     * @var int
+     */
+    protected $id;
```

Comment:
Do these properties really need to be protected? Why not private?

Reply:
They are not private because, they are protected in a parent class